penaltymodel_lp/setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+FACTORY_ENTRYPOINT = 'penaltymodel_factory'  #TODO: import constant
```

Comment:
I don't think there is a nice way to do this because of thew way pip does installs. We would need to import it from `penaltymodel-core` but we can't be sure that it's available on install. `install_requires` is too limited.